### ISSUE TITLE
Log only non-default options in verbose mode

### ISSUE THREAD

**Issue by fabricereix** [Orange-OpenSource/hurl#1927]
Currently:
```
* Options:
*     continue on error: false
*     delay: 100ms
*     follow redirect: false
*     insecure: false
*     max redirect: 50
*     retry: 0
```

After
```
* Options:
*     delay: 100ms
```

**Comment by guilherme-puida** [comment]
I'd like to work on this if possible :^)

**Comment by jcamiel** [comment]
Fixed by #1971
